docs/BuildingEFIenabledEL8bootstrapAMIs.md

```diff
@@ -0,0 +1,347 @@
+          --description '"bootstrap" image for <OS_NAME> 8 (current through YYYY-MM-DD)'
+        ~~~
+
+## Validation
+
+Perform the following setps to ensure that the newly-created AMI(s) are properly configured:
+
+1. Verify desired AMI-attributes:
+
+    ~~~bash
+    aws ec2 describe-images \
+      --query 'Images[].{
+          ID:ImageId,RHUI_Access:UsageOperation,Networking_ENA:EnaSupport,Networking_SRIOV:SriovNetSupport,EFI_Support:BootMode
+        }' \
+      --output table \
+      --image-ids ami-NNNNNNNNNNNNNNNNN
+    ---------------------------------------------------------------------------------------------------
+    |                                         DescribeImages                                          |
+    +----------------+-------------------------+-----------------+-------------------+----------------+
+    |   EFI_Support  |           ID            | Networking_ENA  | Networking_SRIOV  |  RHUI_Access   |
+    +----------------+-------------------------+-----------------+-------------------+----------------+
+    |  uefi-preferred|  ami-NNNNNNNNNNNNNNNNN  |  True           |  simple           |  RunInstances  |
+    +----------------+-------------------------+-----------------+-------------------+----------------+
+    ~~~
+
+    Note: If the `RHUI_Access` value is `RunInstances:0010`, something went wrong in your build workflow. As a result, EC2s launched from the AMI will accrue undesired OS-licensing costs.
+
+2. Perform test-launches of the AMI using both a `t2.medium` and `t3.medium` instance-types and verify that each fully boots up.
+3. Use the CLI to do a quick audit of the instances:
+
+    - Verify correct console-logging setup:
+
+        ~~~bash
+        aws ec2 get-console-output --query 'Output' --output text --instance-id <EC2_ID>
+        ~~~
+
+        Output should be non-null
+
+    - Verify expected attributes:
+
+        ~~~bash
+        aws ec2 describe-instances \
+          --query 'Reservations[].Instances[].{
+              ENA_enabled:EnaSupport,BOOT_Enablement:BootMode,BootMode:CurrentInstanceBootMode
+            }' \
+          --output table \
+          --instance-ids <EC2_IDs>
+        ~~~
+
+        Output should be similar to:
+
+        ~~~bash
+        ----------------------------------------------------------------------
+        |                            DescribeInstances                       |
+        +------------------+---------------+--------------+------------------+
+        |  BOOT_Enablement | BootMode      | ENA_enabled  |    InstanceId    |
+        +------------------+---------------+--------------+------------------+
+        |  uefi-preferred  |  legacy-bios  |  True        | <INSTANCE_1_ID>  |
+        |  uefi-preferred  |  uefi         |  True        | <INSTANCE_2_ID>  |
+        +------------------+---------------+--------------+------------------+
+        ~~~
+
+4. Login to the EC2s and perform a quick audit of the two EC2s. Recommended tests include:
```

Comment:
If possible, can you add examples of what the results for these tests should look like?  Kind of like how you described the expected result of the `billingProducts` test.

Reply:
done